### ISSUE TITLE
Implement some more EDDI events

### ISSUE THREAD

**Issue by alterNERDtive** [alterNERDtive/VoiceAttack-profiles#86]
E.g.

* [ ] https://github.com/EDCD/EDDI/wiki/AFMU-repairs-event
* [ ] https://github.com/EDCD/EDDI/wiki/Asteroid-prospected-event
* [ ] https://github.com/EDCD/EDDI/wiki/Asteroid-prospected-event
* [ ] https://github.com/EDCD/EDDI/wiki/Combat-promotion-event
* [ ] https://github.com/EDCD/EDDI/wiki/Exploration-promotion-event
* [ ] https://github.com/EDCD/EDDI/wiki/Trade-promotion-event
* [ ] https://github.com/EDCD/EDDI/wiki/Empire-promotion-event
* [ ] https://github.com/EDCD/EDDI/wiki/Federation-promotion-event
* [ ] https://github.com/EDCD/EDDI/wiki/Ship-rebooted-event
* [ ] https://github.com/EDCD/EDDI/wiki/Signal-detected-event (might fire often at once and need changes to event handling)
* [ ] https://github.com/EDCD/EDDI/wiki/Surface-signals-detected-event (might fire often at once and need changes to event handling)
* [ ] https://github.com/EDCD/EDDI/wiki/Synthesised-event

**Comment by alterNERDtive** [comment]
The jump count given by `Next jump` could also be used to have RatAttack call the correct jumps automatically by just saying “call jumps”.

**Comment by uchiki** [comment]
"Limpet check" is duplicated when using EDDI's Speech Responder.

**Comment by alterNERDtive** [comment]
> "Limpet check" is duplicated when using EDDI's Speech Responder.

Yes, as the general idea is to _disable_ the standard speech responder. Otherwise you are going to get tons of duplicates, not just that one.

**Comment by uchiki** [comment]
tons?
The problem is the "Limpet check" in ((EDDI undocked)) event.
I want you to set ON / OFF in the same way as AFM and synthesis reports.

**Comment by alterNERDtive** [comment]
> tons?

Yes. E.g. after jumps, or while FSSing.

I’ll look into adding a switch for the limpet check for the next version.

Keep in mind that while this case is simple to do, in general you’re supposed to disable the default speech responders; and if you don’t, you’ll have to sort stuff out yourself :)

**Comment by alterNERDtive** [comment]
Also I just noticed that your request had absolutely nothing to do with this issue, so please keep things on topic.
